session/session.go

```diff
@@ -1504,7 +1504,6 @@ func (s *session) ShowProcess() util.ProcessInfo {
 	tmp := s.processInfo.Load()
 	if tmp != nil {
 		pi = tmp.(util.ProcessInfo)
-		pi.Mem = s.GetSessionVars().StmtCtx.MemTracker.BytesConsumed()
```

Comment:
The `Mem` is not 0 if you run a memory consuming query simultaneously with `SHOW PROCESSLIST`. For example:
```
mysql> SHOW PROCESSLIST;
+------+------+-----------+------+---------+------+-------+-----------------------------------------------------------------+--------+
| Id   | User | Host      | db   | Command | Time | State | Info                                                            | Mem    |
+------+------+-----------+------+---------+------+-------+-----------------------------------------------------------------+--------+
|    1 | root | 127.0.0.1 | tmp  | Query   |    0 | 2     | SHOW PROCESSLIST                                                |      0 |
|    2 | root | 127.0.0.1 | tmp  | Query   |    2 | 2     | insert into tbl select t1.a, t1.b, t1.c from tbl t1 join tbl t2 | 245616 |
+------+------+-----------+------+---------+------+-------+-----------------------------------------------------------------+--------+
2 rows in set (0.02 sec)
```

Reply:
For now, we only trace the memory usage in some memory intensive SQL operators, for example, HashJoin/IndexLookupJoin/IndexLookupReader, etc.

Reply:
Oh, then I prefer removing this `Mem` field since it may confuse users without this knowledge.